scripts/src/public-api-methods.ts

```diff
@@ -223,7 +223,6 @@ export const getPublicAPIMethods = () => {
     "workflows.updateStep",
   ];
 
-  // upcoming platform 2.0 methods we want available but aren't listed quite yet
```

Comment:
Can we keep the comment? That being said it might be out of date

Reply:
Actually you are correct they don't seem to be listed yet, will add comment back